### ISSUE TITLE
Comments on WinterCG CLI API proposal?

### ISSUE THREAD

**Issue by CanadaHonk** [nodejs/TSC#1502]
Hi Node TSCers, I recently started a [CLI API WinterCG proposal](https://github.com/CanadaHonk/proposal-cli-api). This proposes to add a common API between runtimes for CLI functionality, most notably arguments and environment variables (plus more). It would be nice to get some comments/opinions/etc from Node's TSC to help shape the proposal. It is still very early and open to change, but I am hoping to specify a common runtime-agnostic API at some point relatively soon. You can add your own comments on the various points of discussion in [the repo's GH Issues](https://github.com/CanadaHonk/proposal-cli-api/issues) or feel free to discuss/track independently in this issue or somewhere else. Thank you!

**Comment by aduh95** [comment]
That seems more an issue for Node.js collaborators than the TSC, should we transfer this issue to nodejs/node?

**Comment by CanadaHonk** [comment]
Makes sense to me. Just to clarify, I filed in TSC as currently this issue is more for spec comments and if an implementation be accepted into Node in the future, rather than a request to implement it/etc.

**Comment by aduh95** [comment]
FYI Collaborators are the ones who decide what can or cannot go into Node.js, the TSC is involved only if no consensus can be found among Collaborators. Let me transfer the issue then.
